### ISSUE TITLE
Fable testing requests

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1340]
## Description
As a Notify team member, I need to be able to have detailed a11y stories so that I can fix them. 

WHY are we doing?
Some a11y issues identified need
- To be QA'd or validated by real users
- To be more detailed before we start fixing them, as the issues and assistive tech experience might need more documentation or points of view. 

WHAT are we building?
- 5 request to be submitted to Fable testers. These should be unmoderated: compatibility test or self-guided task

VALUE created by our solution
- Refine currently open a11y issues

## Documentation and Artifacts
- [Github a11y board](https://github.com/orgs/cds-snc/projects/31/views/1)
- [Story refinement session](https://docs.google.com/document/d/1PME5NW5MpCKKEYf8mICsjvv7830QofziqPmeKWuys9g/edit)

## Acceptance Criteria
Given some context, when (X) action occurs, then (Y) outcome is achieved 

- [x] Go through short list of a11y issues
- [x] Come up with 5 tasks and tests
- [x] Submit requests through Fable
- [x] Create a new project or rename to Notify?
- [x] Insert feedback and reports into respective a11y stories. 
- [x] Create prototypes for fable testing

* A11y
* Bilingualism
* Privacy considerations
* Security controls in place
* Measuring success and metrics

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (Ping Adrianne to add stuff here or find stuff yourself)

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1340?planning-poker) with Zenhub @andrewleith

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1340?planning-poker) with Zenhub @adriannelee

**Comment by yaelberger-commits** [comment]
Meeting to discuss what the 5 slots should be and which type of testing we should do for each today with Adrianne, Phil, Andrew and Yael

**Comment by amazingphilippe** [comment]
Drafted the 5 requests https://docs.google.com/document/d/16N_JKDlU3q6IRUp94VweCHboCLi8Q_49QxVAYDrmuL4/edit?usp=sharing

**Comment by yaelberger-commits** [comment]
Phil will explore if we can save drafts in Fable for requests. Must get requests in by Sept 15

**Comment by amazingphilippe** [comment]
Fable request drafts entered in Fable. 
Andrew created 2 coded demos. A few tweaks needed, then all good to send the requests.

I need some help from Adrianne, Melissa or Yedida with the GNCTV-15 request. To make sure we get the answers we need

**Comment by yaelberger-commits** [comment]
Will submit the final request today to Fable on testing privacy statements

**Comment by amazingphilippe** [comment]
Fable Requests are submitted. Waiting for them to complete. Usually pretty fast.

**Comment by adriannelee** [comment]
Phil reviewed one of the tests with Andrew. Watch party activity planned to crowdsource next steps.

**Comment by amazingphilippe** [comment]
Finished importing and tagging results. Next step is organizing and grouping things together.

**Comment by amazingphilippe** [comment]
Created insights in Dovetail: https://cds-snc.dovetailapp.com/projects/1vmt3pa4DVI2xRUPHeklAA/v/3FtIuaiIDWSm2gjQv8GPqf/present

**Comment by amazingphilippe** [comment]
Added comments linking to Dovetail in the respective a11y stories. Some stories were already closed? But they were for non-issues. So I guess its all good. [Spreadsheet we used to map tests](https://docs.google.com/spreadsheets/d/1aSRj09NXk7e6zyVGvAw7-8Y6tQ0wC-zPS0YQMgvN1oA/edit#gid=0)
